README.md

```diff
@@ -77,14 +79,55 @@ Usage
 ```
 16. Click “Apply Policy”.
 
+## Generating a proper IAM role
+Note : Only applies when you want to access Device Farm through an IAM Role.
+
+You need to create one Device Farm access role to access the Device Farm Resource and one User or Role depending upon how you are running Jenkins.
+
+- Creating Device Farm Role.
+1. Log into your AWS web console UI.
+2. Click “Identity & Access Management”.
+3. On the left-hand side of the screen, click “Roles”.
+4. Click “Create Role”.
+5. In the AWS service section select Ec2 and click on Next.
+6. Under the Inline Policies header, search and click the “AWSDeviceFarmFullAccess ” policy.This will give complete device farm access.
+7. Now we need to update the maximum expiration of the credentials to 4hrs.
+8. Go the the summary page of the role and click on edit next to "Maximum CLI/API session duration" and select 4hrs.
```

Comment:
8. Go the the summary page of the role and click on edit next to "Maximum CLI/API session duration" and select 4hrs. This sets the expiration of the session associated with the IAM role to 4 hrs.

Reply:
Changed